### PR TITLE
Document array manipulation routines

### PR DESCRIPTION
**Pull request by bveeramani**

### CODE REVIEW

File: docs/source/api-reference/array-manipulation-routines.rst [hunk 1]
```diff
@@ -1,18 +1,14 @@
...
 
-.. currentmodule:: numpy
+.. currentmodule:: nums.numpy
 
 Basic operations
 ================
 .. autosummary::
    :toctree: generated/
 
+    copyto
```
Comment (briancpark) [inline docs/source/api-reference/array-manipulation-routines.rst:11]: This isn't hyperlinked when I compile docs?

File: docs/source/api-reference/array-manipulation-routines.rst [hunk 2]
```diff
@@ -28,6 +24,7 @@ Transpose-like operations
    :toctree: generated/
 
    swapaxes
+   ndarray.T
```
Comment (briancpark) [inline docs/source/api-reference/array-manipulation-routines.rst:27]: `ndarray.T` doesn't show up at all when I compile locally.

File: docs/source/api-reference/array-manipulation-routines.rst [hunk 3]
```diff
@@ -41,6 +38,13 @@ Changing number of dimensions
    expand_dims
    squeeze
 
+Changing kind of array
+======================
+.. autosummary::
+   :toctree: generated/
+
+   asarray
```
Comment (briancpark) [inline docs/source/api-reference/array-manipulation-routines.rst:46]: `asarray` is not hyperlinked?

File: docs/source/api-reference/array-manipulation-routines.rst [hunk 4]
```diff
@@ -60,14 +64,12 @@ Splitting arrays
...
-=============
+Adding and removing elements
+============================
 .. autosummary::
    :toctree: generated/
 
-   tile
-   repeat
-
+   append
```
Comment (briancpark) [inline docs/source/api-reference/array-manipulation-routines.rst:72]: Same thing for append. Doesn't link...

**Review by briancpark — COMMENTED** [review]
I pulled your fork, checked out out to `array-manipulation-routines` and compiled via `make html`. The hyperlinks don't show, and might be because we don't support them yet, although I thought we pushed `asarray` recently. I would advise against adding functions we don't support yet unless you have a good reason to or plan to implement them in the near future.

If I did something wrong in compiling process, lmk.

File: docs/source/api-reference/array-manipulation-routines.rst (hunk 1 above)
Comment (bveeramani) [reply docs/source/api-reference/array-manipulation-routines.rst:11]: It links if you merge #551

File: docs/source/api-reference/array-manipulation-routines.rst (hunk 3 above)
Comment (bveeramani) [reply docs/source/api-reference/array-manipulation-routines.rst:46]: See above

File: docs/source/api-reference/array-manipulation-routines.rst (hunk 4 above)
Comment (bveeramani) [reply docs/source/api-reference/array-manipulation-routines.rst:72]: See above

File: docs/source/api-reference/array-manipulation-routines.rst (hunk 1 above)
Comment (briancpark) [reply docs/source/api-reference/array-manipulation-routines.rst:11]: Gotcha, thanks for clarifying

**Review by briancpark — APPROVED** [review]
LGTM

File: docs/source/api-reference/array-manipulation-routines.rst (hunk 2 above)
Comment (bveeramani) [reply docs/source/api-reference/array-manipulation-routines.rst:27]: Good catch. I know `BlockArray.T` works, but I'm not sure where it's defined. @elibol do you know? Once I know where it's defined, I can add it to the docs

**Pull request merged**